src/node_messaging.cc

```diff
@@ -890,8 +879,8 @@ static void InitMessaging(Local<Object> target,
 
   target->Set(context,
               env->message_port_constructor_string(),
-              GetMessagePortConstructor(env, context).ToLocalChecked())
-                  .Check();
+              GetMessagePortConstructorTemplate(env)
+                  ->GetFunction(context).ToLocalChecked()).Check();
```

Comment:
Can't we just use `ConstructorBehavior::kThrow` for this?

Reply:
@joyeecheung Yeah, it doesn’t work because that also removes the `prototype` property. I’m adding a comment, though.